### ISSUE TITLE
[1.8.9] Infusion not working-Gelum, Machina, Fabrico

### ISSUE THREAD

**Issue by GWYOG** [TheDrunkMafia/ThaumicInfusion#64]
I've tested all the infusions in TI-1.8.9-4.76, and found that these three infusions seemed not working properly.

1. Gelum
I've waiting for it to froze waters for about an hour, nothing happened. Neither works in TI-1.7.10-hotfix-4.32. 
When I checked out you code, I found that there might be a logical error.
ThaumicInfusion/src/main/java/drunkmafia/thaumicinfusion/common/aspect/effect/vanilla/Gelum.java
line 59's 'return' makes it only detect one block in most of the cases. Basically, only x-10,y-10,z-10 will be detected.

2. Machina
It doesn't work in 1.8.9, but it works in 1.7.10. What I've found is that the redstone dust doesn't connect to the infused block in 1.8.9, it connects in 1.7.10. 

3. Fabrico
Everytime I tried to reproduce the same thing you did in the video, I would only break the block immediately. I don't know what's wrong.
Here is my steps:
(1) Use wand to infuse a dirt with Fabrico aspects in creative mode.
(2) Switch to survival mode, take an obsidian in my hand, and then left click the dirt.
(3) dirt breaks immediately :(
What's wrong?

Only TC-5.1.4 and TI-4.7.6 and JEI-2.18.0.87 are installed with forge 1.8.9-11.15.1.1722.

**Comment by TheDrunkMafia** [comment]
Yeah, I can see all of these actually being my bad coding. Essentially, the current 1.8.9 version is really only a test to see if TI works with other mods. Now that it's in 1.8.9 pack on FTB, I've managed to fix allot of issues with compatibility, to the point where it's small cases with specific mods.

Now ever since 1.7.10, no effect code has actually changed. This is due to two reasons, one: I keep going in and out of developing ti, two: I've been focused on getting the back pipeline working.

So yeah, that's the explanation for why the issues with effects is present. Anywho, with that said, I will be going though each effect and rewriting them to actually resemble something I am proud of ;D

Really, I need to re-think what some of them do, because I feel like I've opened up new possibilities with my mod and I have nothing to show for it

**Comment by GWYOG** [comment]
So...do you mean that these effects will change a lot in the near future?

**Comment by TheDrunkMafia** [comment]
Aye. Most of them are place holders, to stress test the system in place. If I may, when you are in game, what is the TI Block Overhead when in game? It's in the F3 debug screen, I'd like to know for reference.

I'll start redoing each effect, so they are more optimized and work more smoothly in game.

**Comment by GWYOG** [comment]
![2016-01-27_11 18 49](https://cloud.githubusercontent.com/assets/16167715/12602878/e4e2b4fe-c4e7-11e5-8860-17ceafac0bcc.png)
Bad news, when I reloaded my test world this time, all infusions disappeared :( 
I don't really know what this information means. What I could do is only to show you the image. 
Since the infusions all disappered, I couldn't show you a better test data.
In many cases, this digit is green, sometimes it's yellow, and rarely it's red(once 7000+)
